sklearn/linear_model/logistic.py

```diff
@@ -1128,6 +1128,18 @@ class LogisticRegression(BaseEstimator, LinearClassifierMixin,
             In SciPy <= 1.0.0 the number of lbfgs iterations may exceed
             ``max_iter``. ``n_iter_`` will now report at most ``max_iter``.
 
+    Examples
+    --------
+    >>> from sklearn.datasets import load_iris
+    >>> from sklearn.linear_model import LogisticRegression
+    >>> X, y = load_iris(return_X_y=True)
+    >>> clf = LogisticRegression(random_state=0).fit(X, y)
+    >>> clf.score(X, y) # doctest: +ELLIPSIS
+    0.96
+    >>> clf.predict_proba(X[:2,:]) # doctest: +ELLIPSIS
+    array([[8.78...e-01, 1.21...e-01, 1.07...e-05],
```

Comment:
Hmm, I'm OK with it but do we have a better way?

Reply:
Like this?

```
>>> res = clf.predict_proba(X[:2,:])
>>> for row in res:
...     print(', '.join(["%.3f" % v for v in row]))
... 
0.873, 0.127, 0.000
0.677, 0.323, 0.000
```

Reply:
I just want to get rid of ``...e-01`` without introducing extra code.
But it seems difficult so let's keep current version.
@adrinjalali There's similar flake8 error in another example.

Reply:
Fixed the other one as well. Thanks for the review!